src/netstandard/FabActUtil/Microsoft.ServiceFabric.Actors.targets

```diff
@@ -15,13 +15,17 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </ItemGroup>
     <Copy SourceFiles="$(TargetDir)\$(AssemblyName).runtimeconfig.dev.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.runtimeconfig.dev.json" SkipUnchangedFiles="true" />
     <Copy SourceFiles="$(TargetDir)\$(AssemblyName).deps.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.deps.json" SkipUnchangedFiles="true" />
+    <Copy SourceFiles="$(TargetDir)\$(AssemblyName).runtimeconfig.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.runtimeconfig.json" SkipUnchangedFiles="true" />
     <Copy SourceFiles="@(ActorServiceAssemblies)" DestinationFolder="$(FabActUtilWorkingDir)" SkipUnchangedFiles="true" />
     <Copy SourceFiles="@(FabActUtilAssemblies)" DestinationFolder="$(FabActUtilWorkingDir)" SkipUnchangedFiles="true" />
   </Target>
-  <Target Name="_UpdateServiceFabricServiceManifest" AfterTargets="Build" DependsOnTargets="_CopyAssembliesToFabActUtilWorkingDir" Condition=" '$(UpdateServiceFabricManifestEnabled)' == 'true' ">
+  <Target Name="_UpdateRuntimeConfig" AfterTargets="Build" DependsOnTargets="_CopyAssembliesToFabActUtilWorkingDir" Condition=" '$(UpdateServiceFabricManifestEnabled)' == 'true' ">
+    <Exec Command="powershell $(FabActUtilWorkingDir)\FixRuntimeConfig.ps1 -RuntimeConfigFile $(FabActUtilWorkingDir)\FabActUtil.runtimeconfig.json" />
```

Comment:
Running a powershell here, will preclude the builds on linux using dotnet build, the modification needs to be done without OS specific tooling.

Reply:
will fix this.